deploy/terraform-datacommons-website/modules/iam/README.md

```diff
@@ -0,0 +1,32 @@
+# IAM module.
+
+This module provides the GCP resources related to IAM with regards to the Data Commons application suite.
+
+- Creation of GCP workload identity service account.
+- Binding of the above SA to the GCP roles required by the DC website application.
+- Binding of the above SA to the GCP roles required to interact with the storage project.
+
+This module assumes that the GKE cluster will have [workload identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) enabled.
```

Comment:
should the user setup GKE cluster separately?

Reply:
Yes, adde a clarification.